LoginPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import { useHistory } from 'react-router-dom';
+
+const LoginSchema = Yup.object().shape({
+  email: Yup.string().email('Invalid email').required('Required'),
+  password: Yup.string().min(8, 'Too Short!').max(50, 'Too Long!').required('Required'),
+});
+
+const LoginPage = () => {
+  let history = useHistory();
+
+  return (
+    <div className="login-page">
+      <div className="image-section">
+        {/* Your image here */}
+      </div>
+      <div className="form-section">
+        <h1>Login</h1>
```

Comment:
The onSubmit function in Formik should ideally be separated out into its own function for modularity and testability. This function can then be tested independently.